### ISSUE TITLE
Some, not all, Telegram messages are not appearing in Zammad

### ISSUE THREAD

**Issue by irl** [zammad/zammad#4278]
### Infos:

* Used Zammad version: Reports as 5.3.x but specifically 06c5262d0b41815e10e5968adc94556720ea43ad
* Installation method (source, package, ..): source
* Operating system: Debian 11
* Database + version: Postgres 9.6
* Elasticsearch version: Elasticsearch 7.17.3
* Browser + version: Firefox 105.0.1 (64-bit)

### Expected behavior:

* Messages sent to the telegram bot appear in Zammad in a timely fashion

### Actual behavior:

* Some, not all, messages do not appear in Zammad. Periods of time where this happens produce errors in the log files.

### Steps to reproduce the behavior:

* Unclear. It seems to mostly affect users that have not interacted with the bot before but we have not got a reliable method to reproduce the issue.

✅  Yes I'm sure this is a bug and no feature request or a general question.

Logs: [zammad-production.log](https://github.com/zammad/zammad/files/9674197/zammad-production.log)

(Happy to try things out or extract futher debug information given instructions.)

**Comment by MrGeneration** [comment]
`E, [2022-09-29T11:28:29.750668 #1-8430840] ERROR -- : Telegram API has returned the error. (ok: "false", error_code: "403", description: "Forbidden: bot was blocked by the user") (Telegram::Bot::Exceptions::ResponseError)`

Sorry, but this seems to be a technical question.
Please note that this repository is the wrong place for technical questions.
Instead, please refer the Zammad community over at: https://community.zammad.org .

In case this turns out the be a bug and no technical question, we'll transfer the relevant information to this repo!

If you require commercial grade support and are no hosted or support contract customer, you can check out our support contracts here: https://zammad.com/pricing#selfhosted
*If you're paying customer already, please consult your Zammad support for assistance! :-)*

----

The other half looks like a duplicate of #3948
